compiler/src/iree/compiler/Codegen/Common/HoistUnrolledVectorExtractInsertSlice.cpp

```diff
@@ -31,8 +31,9 @@ namespace mlir::iree_compiler {
 ///      vector.extract_stirded_slice op.
 /// Returns failure if it can not find the set from vector unrolling artifacts.
 static FailureOr<SmallVector<vector::ExtractStridedSliceOp>>
-getUnrolledExtractSlices(BlockArgument srcTensor,
-                         SmallVector<vector::InsertStridedSliceOp> insertOps) {
+getUnrolledExtractSlices(
+    BlockArgument srcTensor,
+    SmallVectorImpl<vector::InsertStridedSliceOp> &insertOps) {
```

Comment:
this can probably be just `ArrayRef` if it's not being modified.

Reply:
It can't because we call a non-const method (that should probably be const but isn't)

Reply:
Can you add a TODO, actually?

Reply:
That's a todo for mlir though, but I can take a look later

Reply:
Switched to `MutableArrayRef` and added a comment

Reply:
The variables declared via structured bindings are references with zip functions